### PR TITLE
UI changes added file support, broke tests

### PR DESCRIPTION
**Pull request by Jacquelinet19**
# Reliquery Pull Request
**Thank you for contributing to Reliquery.** 
Before submitting this PR please make sure:

### PR checklist
- [x] Your code builds clean without any errors or warnings
- [x] Code is formatted using the supported linter locally before pushing changes
- [x] You have added unit tests where applicable 
- [ x All tests complete and pass with changes locally
- [ ] Have updated the documentation accordingly.

### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Describe proposed changes
-  Added option to download files
-  Added UI element for files

### CODE REVIEW

File: frontend/index.tsx [hunk 1]
```diff
@@ -120,6 +121,39 @@ const Relic = () => {
         }
     }
 
+    function createFilesTable(metaDatas: MetaData[]) {
+        if (metaDatas.length === 0)
+            return "";
+        else {
```
Comment (ibebrett) [inline frontend/index.tsx:127]: notes for next time. the else is not needed since you returned.

File: frontend/index.tsx [hunk 2]
```diff
@@ -120,6 +121,39 @@ const Relic = () => {
...
+                                {Object.keys(metaDatas[0]).map(key => (
+                                    <Th>{key}</Th>
+                                ))}
+                            </Tr>
+                        </Thead>
+                        <Tbody>
+                            {metaDatas.map(metaData => (
+                                <Tr>
+                                    {Object.keys(metaData).map(key => (
+                                        <Td>{metaData[key as keyof MetaData]}</Td>
```
Comment (ibebrett) [inline frontend/index.tsx:142]: i didnt even know about thei keyof thing!

File: frontend/index.tsx [hunk 3]
```diff
@@ -107,7 +108,7 @@ const Relic = () => {
                         <Tbody>
                             {metaDatas.map(metaData => (
                                 <LinkBox as={Tr}>
-                                    <LinkOverlay isExternal={true}
+                                    <LinkOverlay isExternal={false}
```
Comment (AntiAnt) [inline frontend/index.tsx:111]: I think I left this here we should probably revert this but for the next pr.

**Review by ibebrett — APPROVED** [review]

**Review by AntiAnt — APPROVED** [review]
Looks good thanks

File: frontend/index.tsx (hunk 2 above)
Comment (AntiAnt) [reply frontend/index.tsx:142]: All credit goes to J.Pat

**Pull request merged**